### PR TITLE
[KEYCLOAK-11282] - Properly resolve config resolver

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

**Comment by pedroigor** [comment]
@mhajas Here it is

**Comment by pedroigor** [comment]
Quickstart https://github.com/keycloak/keycloak-quickstarts/pull/172.

File: adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakBaseSpringBootConfiguration.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import org.keycloak.adapters.undertow.KeycloakServletExtension;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Import;
```
Comment (mhajas) [inline adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakBaseSpringBootConfiguration.java]: Remove

**Review by mhajas — CHANGES_REQUESTED** [review]
Thank you for the PR @pedroigor. Looks good to me, there is just one unnecessary import.

**Review by mhajas — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**